paderbox/io/play.py

```diff
@@ -147,9 +153,78 @@ def _repr_html_(self):
                     </tr>
                 </table>
                 """.format(self.name, audio_html)
-        na = NamedAudio(data, rate=sample_rate, **kwargs)
-        na.name = name
-        display(na)
+        audio = NamedAudio(data, rate=sample_rate, **kwargs)
+        audio.name = name
+
+    if display:
+        IPython.display.display(audio)
+    else:
+        return audio
+
+
+class Play:
+    def __init__(self, sample_rate=None, **kwargs):
```

Comment:
Why not copy the signature over here? That would make the class easier to use (autocomplete)

Reply:
Done